recipe-client-addon/test/unit/xpc_head.js

```diff
@@ -17,3 +17,10 @@ if (!extensionDir.exists()) {
   extensionDir.append(EXTENSION_ID + ".xpi");
 }
 Components.manager.addBootstrappedManifestLocation(extensionDir);
+
+// Load mocking/stubbing library, sinon
+// docs: http://sinonjs.org/docs/
+Cu.import("resource://gre/modules/Timer.jsm");
+const self = {}; // eslint-disable-line no-unused-vars
```

Comment:
sinon tries to find a global object in a few different ways, one of which is looking for `self`. It throws an error if you don't have this, and [the other places that load sinon during xpcshell tests](https://dxr.mozilla.org/mozilla-central/search?q=sinon-1.16&=mozilla-central) that I could find all have something similar (That's where I copied this from, in fact).

This could use an explaining comment, though.